mamboms/mamboms/mambomsuser/__init__.py

```diff
@@ -17,9 +17,9 @@ def create_django_user(user_name):
 
 def create_user_profile_for(django_user):
     try:
-        django_user.get_profile()
-    except SiteProfileNotAvailable:
-        print 'Profiles are not enabled for this app'
+        django_user.profile
+    # except SiteProfileNotAvailable:
```

Comment:
If this code is dead best to remove it.

Reply:
Yep this is dead code. (https://code.djangoproject.com/ticket/19975). Removing.